tutorials/kitties-part-01/0-intro.mdx

```diff
@@ -0,0 +1,55 @@
+---
+tags:
+  - pallet
+  - poladot.js
+keywords: [pallet, polkadot-js]
+description:
+  From the substrate node template to a fully functional blockchain that handles kitties NFTs.
+level: intermediate
+duration: 2h
```

Comment:
is that realistic?

Reply:
Do you mean the time? well, compiling alone could take like half an hour. It could take more also depending on how much previous experience someone have.